openpype/settings/entities/schemas/projects_schema/schemas/schema_anatomy_templates.json

```diff
@@ -15,6 +15,11 @@
                     "type": "label",
                     "label": "The list of existing placeholders is available here:<br> https://openpype.io/docs/admin_settings_project_anatomy/#available-template-keys "
                 },
+                {
```

Comment:
I don't think this is the right place for a setting like this. it is very likely to be request for use only in very specific scenarios and should therefore be define by a profile most probably. We've pretty much only seen this need in VFX comp workflows, whereas this applies to all publishes across the board then.

Reply:
OK